liblangutil/SemVerHandler.h

```diff
@@ -31,6 +31,8 @@ namespace dev
 namespace solidity
```

Comment:
The namespace should change now, too.

Reply:
should be hanled in e0de5aa - but please double check.